### PR TITLE
Air Pollution Metrics - Version: 1.0.0

### PR DESCRIPTION
**Pull request by Timbar09**
In this PR, I created the Air Pollution Metric web application

**Changes**

- #1 
- #2 
- #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+If you like this project, give it a ⭐️ and let me know what you like in particular.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+I would also like to appreciate [Flaticon](https://www.flaticon.com/) for the home page hero-image and the flag icons.
```
Comment (brytebee) [inline README.md]: You are required to acknowledge the owner of the original design template.

image

File: README.md
Comment (brytebee) [inline README.md:1]: Kindly add a presentation video required for the approval of this Capstone.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Timbar09👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️
- The app is hosted and the link is included in the README.✔️
- Tests are green and pass.✔️

##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes ♻️ 

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md
Comment (Timbar09) [reply README.md:1]: Thanks for the reminder, @brytebee. Can I see a demo of your project?

Comment (DeliceLydia) [reply README.md:1]: - [ ] Please add a video in the README
- presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video should contain a clear and objective description of the project in professional English. 
- The topic and purpose of the project should be made clear.
- The video should contain a demo of the project, highlighting all the project features.
- The video should contain a highlight about a piece of the code you finds interesting or you are proud of making.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Timbar09, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark: ✔️ ✔️

Hi @Timbar09 ✔️ ✔️

Your project is complete! Great job for making the changes requested by the previous reviewer. please kindly consider the optional changes mentioned🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️
- work well presented✔️
- design well followed✔️
- credit to design owner ✔️ 
- No linter errors✔️

video presentation
-- great job for been brief and going straight to the point with your presentation. Please always give a clear and objective description of the project and how helpful it can be. lastly, nice job for adding different search categories 👏🏾 👏🏾 


-- [optional] please always check your project requirements. you were required to use git-flow on this milestone. see more [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**